apps/parsley/src/components/SidePanel/NavGroup/FilterNavGroup/FilterGroup/index.tsx

```diff
@@ -72,24 +72,12 @@ const FilterGroup: React.FC<FilterGroupProps> = ({
   return (
     <Accordion
       defaultOpen
-      onToggle={({ isVisible }) =>
-        sendEvent({ name: "Toggled Filter", open: isVisible })
-      }
+      onToggle={({ isVisible }) => {
+        sendEvent({ name: "Toggled Filter", open: isVisible });
+        setOpenAccordion(isVisible);
```

Comment:
If a user is editing the filter text and closes the accordion, do you think we should reset the edit state? It currently appears next time the accordion is opened.

I'm okay with either, just wanted to flag!

Reply:
hmm that's a great call out. What do you think about this interaction? 

https://github.com/evergreen-ci/ui/assets/4605522/0d400ee1-0ecc-4355-b59e-1bb349120b2a

Reply:
That looks good to me!

Reply:
Thanks!